SpatialGDK/Documentation/content/how-to-use-schema.md

```diff
@@ -6,45 +6,57 @@
     <br/> ![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/schema-button-full-scan.png)<br/>
     _Image: In the GDK toolbar in the Unreal Editor, select **Schema (Full Scan)**_<br/>
-    <br/>When you select **Schema (Full Scan)**, the GDK automatically iterates through all classes* in your project to generate the schema files, and then updates the [SchemaDatabase]({{urlRoot}}/content/glossary#schemadatabase). <br/>
+    <br/>When you select **Schema (Full Scan)**, the GDK automatically iterates through all classes in your project to generate the schema files, and then updates the [SchemaDatabase]({{urlRoot}}/content/glossary#schemadatabase). <br/>
 
-    Run a full scan the first time you generate schema for your project, and whenever you need to generate schema for classes* that are not currently loaded by the Editor.<br/>For example: You need to select **Schema (Full Scan)** if you didn't generate schema after adding a new Blueprint to your game, and that Blueprint is no longer open in the Editor.<br/><br/>
+    Run a full scan the first time you generate schema for your project, and whenever you need to generate schema for classes that are not currently loaded by the Editor.<br/>For example: You need to select **Schema (Full Scan)** if you didn't generate schema after adding a new Blueprint to your game, and that Blueprint is no longer open in the Editor.<br/><br/>
 
-* **Generate Schema (Iterative)**
+* **Iterative**
 
-    To generate schema for classes* that are currently loaded by the Editor: <br/>
+    To generate schema for classes that are currently loaded by the Editor: <br/>
 
-    Select **Schema** in the [GDK Toolbar]({{urlRoot}}/content/toolbars#buttons). The GDK automatically iterates through classes* that are currently loaded by the Editor, generates the schema files and updates the [SchemaDatabase]({{urlRoot}}/content/glossary#schemadatabase).<br/>
+    Select **Schema** in the [GDK toolbar]({{urlRoot}}/content/toolbars#buttons). The GDK automatically iterates through classes that are currently loaded by the Editor, generates the schema files and updates the [SchemaDatabase]({{urlRoot}}/content/glossary#schemadatabase).<br/>
     ![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/schema-button.png)<br/>
-    _Image: In the GDK toolbar in the Unreal Editor, select **Schema**_<br/><br/>
+    _Image: In the GDK toolbar in the Unreal Editor, select **Schema**_<br/>
 
-As the GDK automatically generates all the schema you need, you do not have to write or edit schema manually when using the GDK.
+### Generating schema via the command line
+You can generate schema via the command line using an Unreal commandlet. This method uses a Full Scan (see [Which method to use](#which-method-to-use)). It's useful as part of an automated build system.
```

Comment:
"full scan" ? (For consistency.)

Reply:
What's the automated build system and where can I find out about it?

Reply:
I think "automated build system" refers to something that the user might be using, outside of SpatialOS/the GDK, so here we're just saying that you might want to use the CLI method if you have an automated build system that you want to integrate it into. (@padhansell is that correct?)

Reply:
Yep that's correct :)